src/Umbraco.Web/Trees/MemberTypeTreeController.cs

```diff
@@ -23,7 +23,7 @@ protected override IEnumerable<TreeNode> GetTreeNodesFromService(string id, Form
         {
             return Services.MemberTypeService.GetAll()
                 .OrderBy(x => x.Name)
-                .Select(dt => CreateTreeNode(dt, Constants.ObjectTypes.MemberType, id, queryStrings, Constants.Icons.MemberType, false));
+                .Select(dt => CreateTreeNode(dt, Constants.ObjectTypes.MemberType, id, queryStrings, dt.Icon, false));
```

Comment:
@prjseal Is there a reason why you are not using the null coalescing here on the icon like you do with the media type? Apologies if I have missed out something..